pkgs/test_core/lib/src/runner/vm/test_compiler.dart

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
```

Comment:
[nit] 2021

Reply:
Well, this PR was created in 2020 (and the commit for this file :P)